src/main/java/org/kuali/kfs/pdp/businessobject/PaymentGroup.java

```diff
@@ -335,9 +335,12 @@ public String getPaymentStatusCodeWithHistory() {
             paymentStatusWithHistory += " (Reissued)";
         }
 
-        // check for stale payments, if one payment detail is stale then they all are
-        // CU customization
-        if (!CRConstants.CLEARED.equalsIgnoreCase(paymentStatus.getCode())) {
+        // CU Customization: KFSPTS-27383 do not add Stale to ACH or WIRE
+        final boolean isAchOrWire = StringUtils.equals(disbursementType.getCode(), PdpConstants.DisbursementTypeCodes.ACH) ||
```

Comment:
Is it possible that disbursementType could ever be undefined (meaning null) so that we would get an exception thrown when attempting the getCode call?